docker-compose.yml

```diff
@@ -0,0 +1,246 @@
+version: '3.8'
```

Comment:
This looks like something you have added?....If so keep it, if not then bin it if it's not used